Runtime/Native/Windows/NativeClient.cs

```diff
@@ -0,0 +1,438 @@
+                    {
+                        File.Create(Path.Combine(crashDirFullPath, "backtrace.json"));
+                    }
+                });
+            }
+        }
+        /// <summary>
+        /// Generate path to Crashpad handler binary
+        /// </summary>
+        /// <returns>Path to crashpad handler binary</returns>
+        private string GetDefaultPathToCrashpadHandler()
+        {
+            const string crashpadHandlerName = "crashpad_handler.dll";
+            const string pluginDir = "Plugins";
+            string architecture = IntPtr.Size == 8 ? "x86_64" : "x86";
+
+            string pluginPath = Path.Combine(pluginDir, architecture);
+            string pluginHandlerPath = Path.Combine(pluginPath, crashpadHandlerName);
+
+            // generate full path to .dll file in plugins dir.
+            return Path.Combine(Application.dataPath, pluginHandlerPath);
+
+        }
+        /// <summary>
+        /// Clean scoped attributes
+        /// </summary>
+        internal static void CleanScopedAttributes()
+        {
+            // cleaning scoped attributes should be skipped when 
+            // Configuration.SendUnhandledGameCrashesOnGameStartup  is set to false
+            // the reason behind this decision is to make sure user change in the configuration
+            // won't left and useless data
+            var attributesJson = PlayerPrefs.GetString(ScopedAttributeListKey);
+            if (!HasScopedAttributesEmpty(attributesJson))
+            {
+                return;
+            }
+            var attributes = JsonUtility.FromJson<ScopedAttributesContainer>(attributesJson);
+            foreach (var attributeKey in attributes.Keys)
+            {
+                PlayerPrefs.DeleteKey(string.Format(ScopedAttributesPattern, attributeKey));
+            }
+            PlayerPrefs.DeleteKey(ScopedAttributeListKey);
+        }
+
+        internal static IDictionary<string, string> GetScopedAttributes()
+        {
+            var attributesJson = PlayerPrefs.GetString(ScopedAttributeListKey);
+            if (!HasScopedAttributesEmpty(attributesJson))
+            {
+                return new Dictionary<string, string>();
+            }
+            var result = new Dictionary<string, string>();
+            var attributes = JsonUtility.FromJson<ScopedAttributesContainer>(attributesJson);
+            foreach (var attributeKey in attributes.Keys)
+            {
+                var value = PlayerPrefs.GetString(string.Format(ScopedAttributesPattern, attributeKey), string.Empty);
+                result[attributeKey] = value;
+            }
+
+            // extend scoped attributes with legacy attributes stored by Backtrace-Unity library in previous versions
+            IDictionary<string, string> legacyAttributes = new Dictionary<string, string>()
+                    {
+                        { MachineUuidKey, "guid" },
```

Comment:
What are these legacy attributes?

Reply:
In 3.5.0 we added some attributes to PlayerPrefs (listed above) and now when we're migrating to full list of attributes we still want to be sure, we are able to upload them when we stored them in previous session. Legacy attributes exists now in case when a game crashed and after update (next launch) we should still send a minidump file with guid, session, machine id etc.

Reply:
Ok. Should we remove this code in 3.7? Should be add a reminder to ourselves?

Reply:
No, because there will be always a chance that someone will migrate from 3.5 to n. I think we should remove it once we will introduce breaking change